games/default/files/xpanes/depends.txt

```diff
@@ -1 +0,0 @@
-default
```

Comment:
This is necessary to keep this file otherwise I cannot start the game.

Reply:
Damn, now I understand my problem! I thought these files were no longer used and were replaced by mod.conf